scripts/install.sh

```diff
@@ -22,21 +22,7 @@ fi
 
 if command_exists socketplane; then
     echo >&2 'Warning: "socketplane" command appears to already exist.'
-    while true; do
-        read -p "Would you like to re-install socketplane (y/n) " yn
-        case $yn in
-            [Yy]* )
-                cleanup
-                break
-                ;;
-            [Nn]* )
-                exit 1
-                ;;
-            * )
-                echo "Please answer yes or no."
-                ;;
-        esac
-    done
+    echo >&2 'Warning: "Please run "socketplane uninstall first'
```

Comment:
The issue is if the READ (in install.sh) happens in the pipe install (i.e., wget xxx | sudo sh ) the script fails with no error messages.  It's a catch 22 here... If we add logic to check unattended vs attended (i.e., STDIN open) and skip the read what would be the default?  Just assume reinstall... We could add the sleep 20 and say please CNTRL-C to exit if you don't want to reinstall. I'm ok with that if you guys agree.